### PR TITLE
Us login auth

### PR DESCRIPTION
**Pull request by Said456**
the session of the user

### CODE REVIEW

File: Sauvegarde de ContactManager php.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+         $statement->execute();
+
+        return $this->pdo->lastInsertId();
+    }
+}
+
+
+ 
+$userId = $this->pdo->lastInsertId();
```
Comment (aldaitzwild) [inline Sauvegarde de ContactManager php.txt:36]: What is this file ????? No useless file in the project !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by Said456** [comment]
i delecte the file sauvegarde ContactManager.php.txt

File: src/View/Login/form.html.twig [hunk 2]
```diff
@@ -6,6 +6,8 @@
 {% endblock %}
 
 {% block content %}
+session_start();
```
Comment (aldaitzwild) [inline src/View/Login/form.html.twig:9]: A session_start in a twig file ?? Seems suspicious to me

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Login/form.html.twig (hunk 2 above)
Comment (Said456) [reply src/View/Login/form.html.twig]: it ok now for the delete

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**